core/hooks/use-clipboard.tsx

```diff
@@ -0,0 +1,22 @@
+export type OnSuccess = () => any;
+export type OnError = (error: Error) => any;
+
+export function useClipboard() {
```

Comment:
Was this intended?

Any reason why we bring back `useClipboard`? This is built-in in docusaurus if I'm not mistaken. Docs page code blocks have it by default and we can reuse the code blocks in react pages as well just like I did here:
[`f95fc5c` (#259)](https://github.com/game-ci/documentation/pull/259/commits/f95fc5c93d58f54b9f084ebea735a60022e09772#diff-42eec9aff99056af77cdb3610b3ad8f2ef6c82b0696123159db1ade3af59acc8R66)

![ScreenShot Firefox Developer Edition 2022-05-17 at 17 31 24@2x](https://user-images.githubusercontent.com/1264761/168913679-fb54d11f-e501-4daf-81ae-2fae204f98f8.png)

See official documentation here:
https://docusaurus.io/docs/markdown-features/code-blocks#usage-in-jsx

Reply:
Yea it's used

![image](https://user-images.githubusercontent.com/20756439/168927906-593ebfce-257a-4ddc-87e8-a6c1f70ec1a3.png)

https://user-images.githubusercontent.com/20756439/168928173-4459e8cf-8cda-4dd9-8f3b-b2a85fee7ea2.mp4